src/modules/transaction/hooks/queries/useTransactionsFromPool.js

```diff
@@ -1,9 +1,10 @@
 import { useInvokeQuery } from 'src/modules/common/hooks';
 
-export const useTransactionsFromPool = ({ options = {}, customConfig = {} } = {}) => {
+export const useTransactionsFromPool = ({ options = {}, address, customConfig = {} } = {}) => {
   const config = {
     data: {
       endpoint: 'txpool_getTransactionsFromPool',
+      params: { address },
```

Comment:
The challenge I see with making the address required in the params is that if there's ever a need to get all transactions in the pool, then it becomes harder to do that with the current change

Reply:
Fixed https://github.com/LiskHQ/lisk-desktop/pull/5522/commits/1ce01cec9f40620f4101e0b733593c0982a15d6d